api-wizard/template.yaml

```diff
@@ -30,30 +27,74 @@ spec:
           title: Version
           type: number
           default: 1
-        domainModels:
-          title: Domain Model Definitions
+    - title: API Owner
+      required:
+        - ownerName
+        - ownerEmail
+        - team
+      properties:
+        ownerName:
+          title: Owner Name
+          type: string
+          description: The name of the person to contact regarding this API
+        ownerEmail:
+          title: Owner e-mail
+          type: string
+          description: The e-mail of the person to contact regarding this API
+        team:
+          title: Team
+          type: string
+          description: The team/division responsible for this API
+    - title: Domain Model Definitions
+      type: array
+      items:
+        title: Domain Model
+        type: object
+        properties:
+          domainModelName:
+            title: Domain Model Name
+            type: string
+          domainModelProperties:
+            title: Properties
+            type: array
+            items:
+              type: object
+              properties:
+                propertyName:
+                  title: Property Name
+                  type: string
+                  description: The name of the property
+                propertyValue:
+                  title: Property Value Type
+                  type: string
+                  description: The type of this property's value
+                  enum: ["string","integer","number","boolean","domain model"]
+              allOf:
+                - if:
```

Comment:
NICE! I was wondering if we could do this since the `integer` value type looks like it needs a `format` specifier, but some other value types don't always need that!